tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1045,22 +1049,24 @@ if (/* Comments */ true /* Comments */) {
 } else {}
 
 
-  if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
-      mb_internal_encoding('ASCII');
-  }~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
+  mb_internal_encoding('ASCII');
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <?php
 // Comment
 if (1) {
     // Comment
     // Comment
     echo "Foo"; // Comment
     // Comment
-} elseif (2) {
+} elseif (2) {// Comment
```

Comment:
Alright, I think this is more PSR-compatible @nicoder. Not sure if there should be a space before the comment?

Reply:
Let's left as is, if somebody have problem with this we can have discussion about this and fix it late

Reply:
Space before comment